[2024-10-08T10:53:50.979+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T10:53:50.990+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_id_3.check_if_table_exists manual__2024-10-08T10:53:48.520521+00:00 [queued]>
[2024-10-08T10:53:50.995+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_id_3.check_if_table_exists manual__2024-10-08T10:53:48.520521+00:00 [queued]>
[2024-10-08T10:53:50.995+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-08T10:53:51.003+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): check_if_table_exists> on 2024-10-08 10:53:48.520521+00:00
[2024-10-08T10:53:51.005+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1212) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T10:53:51.006+0000] {standard_task_runner.py:72} INFO - Started process 1214 to run task
[2024-10-08T10:53:51.007+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_id_3', 'check_if_table_exists', 'manual__2024-10-08T10:53:48.520521+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/dag_id_3.py', '--cfg-path', '/tmp/tmpkw_i5oyl']
[2024-10-08T10:53:51.007+0000] {standard_task_runner.py:105} INFO - Job 234: Subtask check_if_table_exists
[2024-10-08T10:53:51.039+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_id_3.check_if_table_exists manual__2024-10-08T10:53:48.520521+00:00 [running]> on host ea4b7cb1f1d7
[2024-10-08T10:53:51.087+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_id_3' AIRFLOW_CTX_TASK_ID='check_if_table_exists' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T10:53:48.520521+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T10:53:48.520521+00:00'
[2024-10-08T10:53:51.088+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-08T10:53:51.088+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-08T10:53:51.089+0000] {logging_mixin.py:190} INFO - Current task name:check_if_table_exists state:running start_date:2024-10-08 10:53:50.991091+00:00
[2024-10-08T10:53:51.089+0000] {logging_mixin.py:190} INFO - Dag name:dag_id_3 and current dag run status:running
[2024-10-08T10:53:51.089+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T10:53:51.089+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-08T10:53:51.089+0000] {python.py:240} INFO - Done. Returned value was: create_table
[2024-10-08T10:53:51.090+0000] {branch.py:38} INFO - Branch into create_table
[2024-10-08T10:53:51.090+0000] {skipmixin.py:230} INFO - Following branch ('create_table',)
[2024-10-08T10:53:51.098+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2024-10-08T10:53:51.119+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T10:53:51.119+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_id_3, task_id=check_if_table_exists, run_id=manual__2024-10-08T10:53:48.520521+00:00, execution_date=20241008T105348, start_date=20241008T105350, end_date=20241008T105351
[2024-10-08T10:53:51.125+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-08T10:53:51.125+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-08T10:53:51.125+0000] {logging_mixin.py:190} INFO - Dag name:dag_id_3 queued_at:2024-10-08 10:53:48.531987+00:00
[2024-10-08T10:53:51.126+0000] {logging_mixin.py:190} INFO - Task hostname:ea4b7cb1f1d7 operator:BranchPythonOperator
[2024-10-08T10:53:51.147+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T10:53:51.166+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-08T10:53:51.166+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
